### ISSUE TITLE
1.4.2 not working

### ISSUE THREAD

**Issue by shinobiclan** [UnamSanctam/SilentXMRMiner#335]
Hi, the version 1.4.2 is not working, I do not know why

the previous 1.4.1 one works fine
Thank you

**Comment by shinobiclan** [comment]
here are the settings

![main](https://user-images.githubusercontent.com/35500797/125868078-4dfa3b8f-3dc2-412a-9012-f9ea2619a7a2.jpg)
![mining](https://user-images.githubusercontent.com/35500797/125868085-d3f67cbb-e94a-483e-bea8-b6ce4665ed58.jpg)
![advances options](https://user-images.githubusercontent.com/35500797/125868092-1a85ae94-6552-4a9a-88e7-6dfbf8d7578e.jpg)

**Comment by UnamSanctam** [comment]
And what doesn't work? Can you enable "DEBUG" and see if you get any error messages?

**Comment by shinobiclan** [comment]
miner build does not work

builder works fine

**Comment by shinobiclan** [comment]
maybe the build does not download or launch the miner

**Comment by UnamSanctam** [comment]
I mean what part doesn't work? If you have Process Hacker running you should see when it stops working. Also, enable "DEBUG" to get error messages if there are any errors.

**Comment by shinobiclan** [comment]
included «DEBUG»,
1.4.2 collects the miner normally

only after starting the build there is no response
I expected about 5-20 minutes
the miner did not appear in the proxy connection logs

the previous version 1.4.1 connects after 1-2 minutes

**Comment by shinobiclan** [comment]
checked through the command output

`tail -f proxy.log | grep LEGION`

**Comment by UnamSanctam** [comment]
Check with Process Hacker, you can scroll to the bottom and see what starts when you run it. You could also search for `--cinit` in the top-right to see if any miners are running.

**Comment by shinobiclan** [comment]
the function {%COMPUTERNAME%} is quite convenient =)


ok, I'll take a look through the hacker process

**Comment by shinobiclan** [comment]
watched through the process hacker

no process

**Comment by shinobiclan** [comment]
the point is most likely that the build does not download or launch the miner

the previous version works fine

**Comment by UnamSanctam** [comment]
If it doesn't then it should say so with "DEBUG" enabled, it also works for me so it would be something with either your computer or your build. Can you send your file here?

**Comment by shinobiclan** [comment]
https://mega.nz/file/vbpUxJRZ#KYkbVOQZM-hGJY5vtPQkJ7OEZ8GrfuRkVGTgi9Sz_LA

I checked it on other devices, it did not work for me
I honestly don't know why
here is the build

**Comment by UnamSanctam** [comment]
You named the file services.exe while having "Install to System32" enabled, there already exists a services.exe from Windows in that folder so it won't be able to "Install" itself with that name.

**Comment by shinobiclan** [comment]
aa, that's it, got it
filenames may conflict
got it
thanks =)aa, that's it, got it
filenames may conflict
got it
thanks =)

**Comment by shinobiclan** [comment]
and here's another question

{% COMPUTERNAME%} / {% USERNAME%} / LEGION

can multiple values be output?

**Comment by UnamSanctam** [comment]
Yes you can do that, it will replace all of those strings.

**Comment by shinobiclan** [comment]
but let's output

desktop name / username / and amount of RAM?

About
how will it be correct?

**Comment by UnamSanctam** [comment]
Well, there is no replacement string for RAM but you don't really need it anyways since it can only use 2GB and if there isn't enough (which there probably is) it will just not use huge pages (what uses 2GB). But for computer name / username you can just do what you sent before: `{%COMPUTERNAME%}/{%USERNAME%}`

**Comment by shinobiclan** [comment]
understood thanks
I will not bother you anymore for today
probably =)

I think the topic can be closed

tried it, it turned out like this:
"CDESKTOPSO/{%USERNAME%}/LEGION"

**Comment by UnamSanctam** [comment]
Yeah that should work, glad we found the issue. I will close it and if there is anything else you can write it here or open a new issue.
